### ISSUE TITLE
Feature: Setup screen readers page

### ISSUE THREAD

**Issue by michaeldev6** [michaeldev6/dev-a11y#12]
- Page lists out major screen readers
  - NVDA
  - JAWS
  - VoiceOver
  - TalkBack
- With the focus of this site being fro developers, should give tips and ideas on how to utilize these SRs for testing. Common shortcuts and how to navigate around with each screen reader. What to listen for, and maybe odd little quirks for each one. 
- Possibly have each screen reader with it's own page? Maybe needed since each one can be quite extensive. But the focus should be on helping developers and "testing" websites with screen readers.
- Also should give an overall explanation of how screen readers view and read through the page.
